avm/res/healthcare-apis/workspace/fhirservice/main.bicep

```diff
@@ -0,0 +1,399 @@
+  'PATCH'
+  'POST'
+  'PUT'
+])
+@description('Optional. Specify the allowed HTTP methods.')
+param corsMethods array?
+
+@description('Optional. Specify how long a result from a request can be cached in seconds. Example: 600 means 10 minutes.')
+param corsMaxAge int?
+
+@description('Optional. Use this setting to indicate that cookies should be included in CORS requests.')
+param corsAllowCredentials bool = false
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. The diagnostic settings of the service.')
+param diagnosticSettings diagnosticSettingType
+
+@description('Optional. The name of the default export storage account.')
+param exportStorageAccountName string?
+
+@description('Optional. The name of the default integration storage account.')
+param importStorageAccountName string = ''
+
+@description('Optional. If the import operation is enabled.')
+param importEnabled bool = false
+
+@description('Optional. If the FHIR service is in InitialImportMode.')
+param initialImportMode bool = false
+
+@description('Optional. The lock settings of the service.')
+param lock lockType
+
+@description('Optional. Array of role assignments to create.')
+param roleAssignments roleAssignmentType
+
+@allowed([
+  'Disabled'
+  'Enabled'
+])
+@description('Optional. Control permission for data plane traffic coming from public networks while private endpoint is enabled.')
+param publicNetworkAccess string = 'Disabled'
+
+@allowed([
+  'no-version'
+  'versioned'
+  'versioned-update'
+])
+@description('Optional. The default value for tracking history across all resources.')
+param resourceVersionPolicy string = 'versioned'
+
+@description('Optional. A list of FHIR Resources and their version policy overrides.')
+param resourceVersionOverrides object?
+
+@description('Optional. If the SMART on FHIR proxy is enabled.')
+param smartProxyEnabled bool = false
+
+@description('Optional. The managed identity definition for this resource.')
+param managedIdentities managedIdentitiesType
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+var formattedUserAssignedIdentities = reduce(
+  map((managedIdentities.?userAssignedResourceIds ?? []), (id) => { '${id}': {} }),
+  {},
+  (cur, next) => union(cur, next)
+) // Converts the flat array to an object like { '${id1}': {}, '${id2}': {} }
+
+var identity = !empty(managedIdentities)
+  ? {
+      type: (managedIdentities.?systemAssigned ?? false)
+        ? (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned')
+        : (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'UserAssigned' : 'None')
+      userAssignedIdentities: !empty(formattedUserAssignedIdentities) ? formattedUserAssignedIdentities : null
+    }
+  : null
+
+var accessPolicies = [
+  for id in accessPolicyObjectIds ?? []: {
+    objectId: id
+  }
+]
+
+var exportConfiguration = {
```

Comment:
The var is only used in a single location. Should not be a var

Reply:
Removed the variable: https://github.com/Azure/bicep-registry-modules/pull/1467/commits/277fb756d1b61515cc9afafbd3b6a13315ea75fe